redux/redux.d.ts

```diff
@@ -1,52 +1,81 @@
-// Type definitions for Redux v1.0.0
-// Project: https://github.com/rackt/redux
-// Definitions by: William Buchwalter <https://github.com/wbuchwalter/>, Vincent Prouillet <https://github.com/Keats/>
+// Type definitions for Redux v3.3.1
+// Project: https://github.com/reactjs/redux
+// Definitions by: Jakub Strojewski <https://github.com/ulfryk>
 // Definitions: https://github.com/borisyankov/DefinitelyTyped
 
 declare module Redux {
 
-    interface ActionCreator extends Function {
-        (...args: any[]): any;
+    // Interfaces
+
+    interface IAction {
```

Comment:
Not using capital `I` makes all those types totally misleading. It is common standard to name JS constructors (now classes) with PascalCase. So `I` in front makes it easy to know if its concrete Type or just ambient interface. It is quite important when dealing with reflection metadata like this:

```
import { HttpHelper } from 'helpers';

class SomeService {
    constructor(
        @Inject() public helper: HttpHelper
    ) {}
}
```

If HttpHelper was interface, ambient class or type - it will not work. So adding `I` for all interfaces describing entities without constructor makes it much more useful. That's my point.

I also consider renaming `Store<S>` to `IStore<S>` as there is no class/constructor for this "type".